cmd/local-gadget/utils/trace.go

```diff
@@ -61,9 +62,9 @@ func RunTraceAndPrintStatusOutput(config *TraceConfig, customResultsDisplay func
 
 	operations := localGadgetManager.ListOperations(traceName)
```

Comment:
Perhaps modify `ListOperations` to return `[]gadgetv1alpha1.Operation` instead of `[]string` to avoid so many castings?